repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletResponseWrapper.scala

```diff
@@ -223,6 +225,10 @@ class HttpServletResponseWrapper(originalResponse: HttpServletResponse, headerMo
     addHeader(name, DateUtils.formatDate(new Date(timeSinceEpoch)))
 
   override def addHeader(name: String, value: String): Unit = {
```

Comment:
It seems like we may as well. We aren't actually changing behavior, just notifying of unexpected behavior. On a custom logger made just for that purpose, even. Is that ever a bad thing?